bulk_grades/api.py

```diff
@@ -345,6 +345,16 @@ def validate_row(self, row):
         if row['course_id'] != self.course_id:
             raise ValidationError(_('Wrong course id {} != {}').format(row['course_id'], self.course_id))
 
+    def status(self):
+        result = super().status()
+        result['error_messages'] = [
```

Comment:
Two things:
- Is this readable?
- It is possible to have this on `super-csv` but I think this change is much smaller and work just as fine. Currently, `super-csv` isn't sending line number to the front end.

Reply:
I don’t think this should be here. move this to the view, I think.